packages/react/src/components/table/table-row.tsx

```diff
@@ -107,31 +107,6 @@ export const StyledTableRow = styled.tr<StyledTableRowProps>`
         }
     `}
 
-    ${({ $error, theme }) => $error && css`
-        position: relative;
-
-        &::after {
```

Comment:
Buh bye, you won't be missed.

Ca fix le bug qu'on ne peut pas clicker dans le row quand il y a un erreur.

Reply:
ahha yep
ça empechait aussi de copier le texte de la row